README.md

```diff
@@ -21,17 +18,114 @@ To toggle between the Phase 5 transition and post-transition modes we have defin
 #### Transition
 <pre>
 sm2 --start CTC_TRADERS_P5_ACCEPTANCE_TRANSITION
-sm2 --stop CTC_DEPARTURE_TRADER_DETAILS_FRONTEND_TRANSITION
+sm2 --stop MANAGE_TRANSIT_MOVEMENTS_DEPARTURE_FRONTEND_TRANSITION
 sbt -Dplay.additional.module=config.TransitionModule run
 </pre>
 
 #### Final
 <pre>
 sm2 --start CTC_TRADERS_P5_ACCEPTANCE
-sm2 --stop CTC_DEPARTURE_TRADER_DETAILS_FRONTEND
+sm2 --stop MANAGE_TRANSIT_MOVEMENTS_DEPARTURE_FRONTEND
 sbt -Dplay.additional.module=config.PostTransitionModule run
 </pre>
 
+### Feature toggles
+
+The following features can be toggled in [application.conf](conf/application.conf):
+
+| Key                          | Argument type | sbt                                                             | Description                                                                                                                                                                                    |
+|------------------------------|---------------|-----------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `trader-test.enabled`        | `Boolean`     | `sbt -Dtrader-test.enabled=true run`                            | If enabled, this will override the behaviour of the "Is this page not working properly?" and "feedback" links. This is so we can receive feedback in the absence of Deskpro in `externaltest`. |
+| `banners.showUserResearch`   | `Boolean`     | `sbt -Dbanners.showUserResearch=true run`                       | Controls whether or not we show the user research banner.                                                                                                                                      |
+| `play.additional.module`     | `String`      | `sbt -Dplay.additional.module=config.PostTransitionModule run`  | Controls which module (TransitionModule or PostTransitionModule) we bind to the application at start-up.                                                                                       |
+| `play.http.router`           | `String`      | `sbt -Dplay.http.router=testOnlyDoNotUseInAppConf.Routes run`   | Controls which router is used for the application, either `prod.Routes` or `testOnlyDoNotUseInAppConf.Routes`                                                                                  |
+
+### Running Scaffold
+
+Create New Page in sbt:
```

Comment:
For the departure sub microservices I'd be tempted to just have a link to the manage-transit-movements-departure-frontend README

Reply:
Yeah, that will make it easier to manage if any of that information does chance in the future, pushed an update